src/components/LoginRegister.js

```diff
@@ -0,0 +1,234 @@
+import {
+  VStack,
+  Heading,
+  Button,
+  Box,
+  FormControl,
+  InputGroup,
+  InputLeftElement,
+  InputRightElement,
+  Input,
+  FormHelperText,
+  FormLabel,
+} from '@chakra-ui/react';
+import { useState, useRef } from 'react';
+
+const LoginRegister = (props) => {
+  const [showPassword, setShowPassword] = useState(false);
+  const filePicker = useRef(null);
+  const handleShowPasswordChange = () => {
+    setShowPassword(!showPassword);
+  };
+  return (
+    <VStack w="100%" h="100vh" mx="left" bgGradient="linear(to-r, #E8F6EF, #E8F6EF)">
+      {props.activeOption === 'login' ? (
+        <Box bg="white" mt="10%" textAlign="center" boxShadow="md" p={6} rounded="md">
+          <Heading color="#2FE1D6" mb="8" textShadow="1px 1px #e0e0e0">
+            Welcome
+          </Heading>
+          <form
+            onSubmit={(e) => {
+              e.preventDefault();
+              props.setIsLoggedIn(true);
+            }}
+          >
```

Comment:
For function that have more than 1 line, please extract to a separate function
```suggestion
          <form
            onSubmit={(e) => handleLogin(e)}
          >
```

and extract the function above like


```js
function handleLogin (e) {
    e.preventDefault();
    props.setIsLoggedIn(true);
}
```